commands/seed.js

```diff
@@ -20,16 +25,149 @@ const USERS = [
 const F1_2018_SEASON = 'F1-2018';
 const F1_CHAMPIONSHIP = 'F1';
 const F1_2018_TEAMS = [
```

Comment:
I know that is just a seed, but I would put in another file to keep cleaner this

Reply:
To be honest... this file is meant to be so temporal that I don't think you refactor is worth the effort, bear in mind that as soon as we launch racermanager, we will be using the admin.